html/php-pre-challenge3/index.php

```diff
@@ -1,6 +1,70 @@
 <?php
+
+// URLで入力されるtarget値$limitに代入している
 $limit = $_GET['target'];
 
+// db接続情報
 $dsn = 'mysql:dbname=test;host=mysql';
 $dbuser = 'test';
 $dbpassword = 'test';
+
+// dbへ接続と接続できなかった際の表示を設定
+try {
+  // pdoクラスのインスタンス化
+  $pdo = new PDO($dsn, $dbuser, $dbpassword);
+} catch (PDOException $e) {
+  echo 'DBに接続できていません' . $e->getMessage();
```

Comment:
```suggestion
  $errMsg = 'DBに接続できていません:' . $e->getMessage();
  echo json_encode($errMsg);
  die();
```
こうしないとエラーメッセージをレスポンス送れないですね。json_encodeとdieがポイントです。dieはexitでもよいです。

Reply:
b065957 こちらのコミットでjson形式のレスポンスに変更しました。